### DISCUSSION TITLE
Placing a model on the "ground" possible? Non AR.

### DISCUSSION THREAD

**Discussion by brianyuen** [google/model-viewer#3359] [Q&A] [answered]
Hello,
I'm using model-viewer in the hopes of placing a vehicle that sits on the floor. Not referring to AR mode, but just a click and drag to explore. Currently the model seems to float above its environment. I would like to [achieve something like this](https://imgur.com/a/lS85kTk). Is this even remotely possible with model-viewer?

Thanks!

**Comment by elalish** [comment, marked as answer]
The closest we have at the moment is turning on shadows, which creates a floor plane. We are thinking of building an API to make proper 3D background scenes possible though.
